### PR TITLE
opencv: add missing static libs

### PR DESCRIPTION
**Pull request by jeroen**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

I'm terribly sorry, my PR earlier today I overlooked that opencv also builds a few 3rd party libraries that are needed to statically link an application. 

This PR adds the missing `libippicv.a`, `libippiw.a`, and `libittnotify.a` to the bottle.

### CODE REVIEW

File: Formula/opencv.rb [hunk 1]
```diff
@@ -98,6 +98,8 @@ def install
       system "cmake", "..", "-DBUILD_SHARED_LIBS=OFF", *args
       system "make"
       lib.install Dir["lib/*.a"]
+      lib.install Dir["3rdparty/lib/*.a"]
```
Comment (ilovezfs) [inline Formula/opencv.rb]: Does `lib.install Dir["3rdparty/lib/**/*.a"]` grab them all?

Comment (ilovezfs) [reply Formula/opencv.rb]: or even `lib.install Dir["3rdparty/**/*.a"]`

Comment (jeroen) [reply Formula/opencv.rb]: Let me test that.

**Comment by jeroen** [comment]
Thank you for the suggestion @ilovezfs. It works, and I have updated the PR.

**Pull request closed without merging**